From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@murdockcrc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@murdockcrc - Thanks for the fix.

iainfoulds committed:
[VMSS] Edits and revisions to upgrade doc

iainfoulds committed:
[VMSS] Fixing link validation errors

gsacavdm committed:
Add syntax highlighting to code blocks

JeffGoldner committed:
Update with alt text, add missing or new items

ausfeldt committed:
Add 1.1.1 to release notes

@mimig1

femsulu committed:
(Forums Team) fix typo #6326

changed "Front -nd" to "Front-end"

mimig1 committed:
Cosmos DB: Updating old videos to new videos

sanjeev3 committed:
Update image-moderation-api.md

sanjeev3 committed:
Update image-moderation-quickstart-dotnet.md

mattbriggs committed:
adding white backgrounds to images for dark theme.

Commit:
Update owner

Commit:
Update owner

Commit:
Update owner

Commit:
Update owner

Commit:
Update owner

Commit:
Update owner

Commit:
Update author

gsacavdm committed:
Update variations for Azure Functions

douglaslMS committed:
Revised out-of-date article, updated screenshots, per PM.

subsarma committed:
Updated PS install instructions

douglaslMS committed:
Resized new screenshots to 780 px max width.

gsacavdm committed:
Update gov-services - functions has UI now

PRMerger13 committed:
Merge pull request #36383 from gsacavdm/patch-2

Update variations for Azure Functions

ausfeldt committed:
fix version copy paste typos

@mimig1

PRMerger15 committed:
Merge pull request #36385 from subsarma/patch-1

Updated PS install instructions

douglaslMS committed:
Fixed an escaping issue with angle brackets.

gsacavdm committed:
Remove Azure functions quickstart

Remove Azure functions quickstart since we now have the Azure Functions UI and the regular Azure Functions quickstarts from public Azure are good for Gov.

iainfoulds committed:
[VMSS] Updating PowerShell release version and updating redirects

GitHubber17 committed:
Merge pull request #36375 from mattbriggs/mdb_addbackground

adding white backgrounds to images for dark theme.

craigshoemaker committed:
Change owner from tamram to craigshoemaker

iainfoulds committed:
[VMSS] Correcting landing page links

GitHubber17 committed:
Merge pull request #36373 from sanjeev3/master

Updates to image moderation articles.

GitHubber17 committed:
Merge pull request #36378 from Jann-Skotdal/master

Update code author

Commit:
Merge pull request #36350 from femsulu/patch-7

(Forums Team) fix typo #6326

rjagiewich committed:
Merge pull request #36387 from gsacavdm/patch-1

Add syntax highlighting to code blocks

GitHubber17 committed:
Merge pull request #36389 from iainfoulds/vmssposhupdate

[VMSS] Updating PowerShell release version and updating redirects

GitHubber17 committed:
Merge pull request #36299 from ausfeldt/patch-20

Add 1.1.1 to release notes

GitHubber17 committed:
Update author-visually.md

GitHubber17 committed:
Merge pull request #36384 from gsacavdm/patch-3

Update gov-services - functions has UI now

gsacavdm committed:
Remove gov-functions doc & imgs

rwike77 committed:
fixed merge conflict

GitHubber17 committed:
Merge pull request #36386 from douglaslMS/adf-author-visually

 Revised out-of-date article, updated screenshots, per PM.

GitHubber17 committed:
Merge pull request #36390 from craigshoemaker/crs-article-reassignment

Change owner from tamram to craigshoemaker

GitHubber17 committed:
Merge pull request #36388 from gsacavdm/patch-2

Remove Azure functions quickstart

Commit:
Merge pull request #36393 from rwike77/deployfix

fixed merge conflict

mikeurnun committed:
fixes #5684

fixes MicrosoftDocs/azure-docs#5684

douglaslMS committed:
Fixed AAD reference per PR reviewer comment.

Commit:
remove small bit of text

tfitzmac committed:
added event grid template samples

PRMerger14 committed:
Merge pull request #36396 from douglaslMS/adf-fix-aad

Fixed AAD reference per PR reviewer comment.

cherylmc committed:
newarticle

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5304

mimig1 committed:
Acrolink fixes

mimig1 committed:
Corrected format of video links

iainfoulds committed:
[VMSS] Incorporating PM feedback

PRMerger18 committed:
Merge pull request #36401 from craigcaseyMSFT/v-craic0327

remove small bit of text

rmca14 committed:
Merge pull request #36398 from tfitzmac/0327egsamples

added event grid template samples

iainfoulds committed:
[VMSS] Adding 'Next steps' section

rmca14 committed:
Merge pull request #36400 from iainfoulds/vmdockerextension

[VM] Docker VM extension

Commit:
Added clarity for getting DNS suffix for NIC.

iainfoulds committed:
Update dockerextension.md

iainfoulds committed:
[VMSS] Updating code samples for consistency

PRMerger10 committed:
Merge pull request #36403 from jimdial/patch-118

Added clarity for getting DNS suffix for NIC.

tdykstra committed:
typo

msmbaldwin committed:
Update azure-health.md

PRMerger12 committed:
Merge pull request #36404 from iainfoulds/patch-1

[VM] Docker VM extension

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

PRMerger14 committed:
Merge pull request #36407 from msmbaldwin/patch-3

Update azure-health.md

mimig1 committed:
fixed format in serverless video

PRMerger15 committed:
Merge pull request #36408 from tdykstra/patch-27

typo

vhorne committed:
Update from public PR 5339 plus more

ktoliver committed:
Merge pull request #36274 from JeffGoldner/master

Update with alt text, add missing or new items

ktoliver committed:
Merge pull request #34421 from iainfoulds/vmssupgrade

[VMSS] Edits and revisions to upgrade doc

PRMerger20 committed:
Merge pull request #36411 from vhorne/pubpr5339

Update from public PR 5339 plus more

cherylmc committed:
Update vpn-gateway-troubleshoot-point-to-site-osx-ikev2.md

Commit:
updating author

Ja-Dunn committed:
Merge pull request #36399 from cherylmc/tsx

new article

roygara committed:
Updating article owners.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updateOwner

Commit:
Updated

PRMerger13 committed:
Merge pull request #36395 from mike-urnun-msft/patch-3

fixes #5684

PRMerger15 committed:
Merge pull request #36421 from roygara/updateOwner

Updating additional article owners.

PRMerger17 committed:
Merge pull request #36413 from jeffgilb/327.1

updating author

mimckitt committed:
Fixed error in first example

The location parameter was specified twice: 

New-AzureRmServiceFabricCluster -ResourceGroupName $resourceGroupName -Location $resourceGroupLocation -CertificateOutputFolder $certOutputFolder -CertificatePassword $certpassword -CertificateSubjectName $CertSubjectName -OS $os -VmPassword $vmpassword -VmUserName $vmuser -Location $ResourceGroupName

Ja-Dunn committed:
Merge pull request #36362 from mimig1/03272018videos

Cosmos DB: Updating old videos to new videos

v-alje committed:
Resolving merge conflict

Commit:
Merge pull request #36426 from MicahMcKittrick-MSFT/patch-9

Fixes MicrosoftDocs/azure-docs#6372

v-alje committed:
Merge pull request #36427 from MicrosoftDocs/FromPublicRepo

From public repo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
